### ISSUE TITLE
Some menu types need adjustements

### ISSUE THREAD

**Issue by rich20** [Kunena/Kunena-Forum#9125]
# Menu item types:

## Custom Content
**The field for the content is missing**
![Bildschirmfoto vom 2022-06-20 17-08-45](https://user-images.githubusercontent.com/888847/174637519-cf090a73-5431-46db-8346-9ab243aee8a0.png)
<br />
## Recent Topics
**The selection option is missing and also the category selection**
![Bildschirmfoto vom 2022-06-19 17-15-544](https://user-images.githubusercontent.com/888847/174639640-b3ce4053-fbf1-4895-9341-d50ad05ac2cf.png)
<br />
## My Topics
**The selection option is missing and also the category selection**
![Bildschirmfoto vom 2022-06-19 17-15-545](https://user-images.githubusercontent.com/888847/174641510-8880c581-f975-497e-a85f-df94c319bf1e.png)
<br />
## Recent Posts
**The selection option is missing and also the category selection**
![Bildschirmfoto vom 2022-06-19 17-15-546](https://user-images.githubusercontent.com/888847/174642591-856747a5-43f7-4e25-a035-3093377ff5cd.png)
<br /><br />
**In summary:**
- [ ] Custom Content
- [ ] Recent Topics
- [ ] My Topics
- [ ] Recent Posts

**Comment by rich20** [comment]
**Custom Content**
- [x] Is ok in the backend, but does not show the content in the frontend

**Recent Topics & My Topics & Recent Posts**
- [x] It's not possible to choose all the categories that you want to show or hide (see image below)
- [x] In Recent Posts this selection field is missing completely
.
![Bildschirmfoto vom 2022-06-26 20-09-33](https://user-images.githubusercontent.com/888847/175828781-469c39a6-01e7-490e-96a2-bcc00c489448.png)
